### DISCUSSION TITLE
ARK: Survival Ascended

### DISCUSSION THREAD

**Discussion by r0lfi** [GameServerManagers/LinuxGSM#4355] [General]
Will support for ARK: Survival Ascended be added ?

**Comment by gOOvER** [comment]
Its only Windows and atm it do not run in Wine; iso i think; no

**Reply by r0lfi** [reply]
Ah, okay. I didn't know that. Thanks for the fast reply.

**Reply by BazemanX** [reply]
They have opened up for everyone to run servers and there are now many people who got it up and running fine on linux. Several guides on reddit and Steam. Should be possible to fix linux server as well.

**Comment by Zeltrax** [comment]
+1 on this love GSM would love it to do ASA

https://ark.wiki.gg/wiki/Dedicated_server_setup#Linux
